contentcuration/contentcuration/api.py

```diff
@@ -7,9 +7,21 @@
 
 from django.core.files import File as DjFile
 from django.db.models import Q
+from django.http import HttpResponse
 from kolibri.content import models as KolibriContent
 from kolibri.content.utils import validate
 from kolibri.content.api import *
+from django.db import transaction
+import models
+
+def count_files(node):
```

Comment:
Something like `node.get_descendants().exclude(kind_id="topic").count()` will give this to you in a single, fast SQL query.